Gems/Terrain/Code/Source/Components/TerrainWorldRendererComponent.cpp

```diff
@@ -44,8 +44,15 @@ namespace Terrain
 
             serialize->Class<ClipmapConfiguration>()
                 ->Version(1)
+                ->Field("ClipmapSize", &ClipmapConfiguration::m_clipmapSize)
+                ->Field("MacroClipmapMaxRenderRadius", &ClipmapConfiguration::m_macroClipmapMaxRenderRadius)
+                ->Field("DetailClipmapMaxRenderRadius", &ClipmapConfiguration::m_detailClipmapMaxRenderRadius)
```

Comment:
These two should be unnecessary - I thought MacroClipmapMaxRenderRadius should always equal MeshConfiguration::m_renderDistance, and DetailClipmapMaxRenderRadius should always equal DetailMaterialConfiguration::m_renderDistance?
i.e. We always draw macro materials the same distance as meshes, and detail materials draw to the same distance whether or not they use clipmaps.

Reply:
I have removed them and have them connected to the equivalent settings.